WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -0,0 +1,146 @@
+import SwiftUI
+
+struct ThemesPreviewView: View {
+    enum PreviewDevice: CaseIterable {
+        case mobile
+        case tablet
+        case desktop
+
+        static var defaultDevice: PreviewDevice {
+            return UIDevice.current.userInterfaceIdiom == .pad ? .tablet : .mobile
+        }
+
+        var width: CGFloat {
```

Comment:
Nit: Please also add a comment explaining what this width is and how it's calculated.

Reply:
Thanks, great suggestion. At first I just copied this directly from WordPress iOS, but I agree that more explanation will be useful. Added in https://github.com/woocommerce/woocommerce-ios/pull/11452/commits/ef983af950df514134c3b8ff56f2d43c221840d8